client/python/gradio_client/data_classes.py

```diff
@@ -6,7 +6,7 @@
 
 
 class FileData(TypedDict):
-    name: str | None  # filename
+    path: str | None  # filename
     data: str | None  # base64 encoded data
```

Comment:
I left this on the client `FileData` model for the above backwards compat reasons.

Reply:
Let's comment to explain this is only here for backwards compatibility. 

Regarding this, why is `data` needed for backwards compatibility, but renaming `name` -> `path` doesn't break backwards compatibility?

Reply:
@freddyaboulton Can i remove this data field as per the above comment or do I need to add `path` back?

Reply:
This is only used for type hints in the `serializing.py` class, which is only needed for backwards compatibility with v3.

So let's change `path` back to `name` and make it clear this is only used in V3 spaces.